demos/mkfile

```diff
@@ -0,0 +1,31 @@
+APE=/sys/src/ape
+<$APE/config
+
+TARG=\
+	firework\
+	ozdemo\
+	ptest\
+	rain\
+	testcurs\
+	tuidemo\
+	worm\
+	xmas\
+
```

Comment:
misses the newer demos, newtest, speed and possibly picsview

Reply:
speed could not be built since I am missing sys/timeb.h

Reply:
Can you check with an option to build without sys/timeb.h? Maybe @benavento has an idea?

Reply:
sys/timeb.h is required by `ftime()`,  for which the man page says
```
This function is obsolete.  Don't use it... gettimeofday(2)
gives microseconds;...
```
(Which is my fault;  I wrote `speed.c`.)  I'll revise the code to use `gettimeofday()`.

Reply:
Should work now (see commit 0b349973).  Turns out my use of `ftime()` was not totally stupid;  in fact,  that was the only solution that works on some older compilers,  and we can't get rid of it completely.  But it's now only used on those older compilers,  and it should compile/run correctly everywhere,  including Plan9.

Reply:
@staalmannen can you please update from master, add the speed demo and test it on plan9?

Reply:
speed now builds (I got 19 fps). I had to disable wchar.h since that is not present by default, but it seems like nothing in speed.c depended on it.

Reply:
You are correct.  Compiled `speed.c` without `wchar.h` on X11, VT, DOS, DOSVGA,  SDL1,  SDL2,  and WinGUI,  and everything worked.  I've removed that line (commit 348d53f1).

Reply:
Ok great! I will just import that one so that there are no merge conflicts if you decide to take this pull request.